cmd/inspector/main.go

```diff
@@ -103,6 +107,12 @@ var (
 		Short: "dump all nodes in the routing table",
 		RunE:  DumpNodes,
 	}
+	drawTableCmd = &cobra.Command{
+		Use: "routing-graph <node_id>",
+		Short: "Save the routing table graph as a dot file with name routing-graph-<node-id>",
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
Can we have the `<node_id> ` argument optional? When not specified it should draw the routing-graph of the current node.